subprojects/android-test/test-app/src/androidTest/kotlin/com/avito/android/ui/test/InProcessClickButtonsTest.kt

```diff
@@ -75,4 +75,18 @@ class InProcessClickButtonsTest {
 
         Screen.buttons.longClickableContainerIndicator.checks.isVisible()
     }
+
+    @Test(expected = PerformException::class)
+    fun hiddenButton_fails() {
+        Screen.buttons.hiddenButton.click()
+
+        Screen.buttons.hiddenButtonIndicator.checks.isDisplayed()
+    }
+
+    @Test
+    fun animatedView_performs() {
+        Screen.buttons.animatedView.click()
```

Comment:
Before the fix:
![Screenshot from 2020-04-10 14-12-06](https://user-images.githubusercontent.com/1104540/78987049-7edb3600-7b35-11ea-8d89-d1aba2bc3332.png)